.circleci/config.yml

```diff
@@ -328,6 +334,7 @@ workflows:
           airflow_version: 1.10.7
           distribution_name: buster
           docker_repo: astronomerinc/ap-airflow
+          extra_args: "-t astronomerinc/ap-airflow:1.10.7-7-buster"
```

Comment:
Does this actually push up the extra tag too?

Reply:
Given there's 

```
      - publish:
          name: publish-1.10.7-buster
```

I don't think this change alone is enough?

Reply:
And we only do

```
            docker save -o saved-images/<< parameters.image_name >>.tar << parameters.image_name >>
```

so only one image will get loaded back in to bet pushed.

Reply:
Yeah, it won't.  Fixing it

Reply:
Updated. I think this approach is better. And it should work because of https://github.com/astronomer/ap-airflow/blob/2579903fa083808e4950fe7ab2f5b4b0c44a5275/.circleci/config.yml#L837-L841